qlib/CsvUtil.qm

```diff
@@ -657,6 +657,9 @@ public namespace CsvUtil {
             # list of idx to field transformarions, in order of spec
             hash m_resolve_by_idx;
 
+            # fake specs based on the first non-header row
+            bool fakeHeaderNames;
```

Comment:
There are unfortunately many name inconsistencies in CsvUtil.qm (not 
only introduced by me ;-)  It should be polished completely.

Dne 3.8.2016 v 22:32 pavelkveton napsal(a):
>
> In qlib/CsvUtil.qm 
> <https://github.com/qorelanguage/qore/pull/1140#discussion_r73414955>:
>
> > @@ -657,6 +657,9 @@ public namespace CsvUtil {
> >              # list of idx to field transformarions, in order of spec
> >              hash m_resolve_by_idx;
> >
> > +            # fake specs based on the first non-header row
> > +            bool fakeHeaderNames;
>
> can you fix the naming differences, please? there is m_resolve_by_idx, 
> but fakeHeaderNames member...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/qorelanguage/qore/pull/1140/files/be0a8a9a7532c8e30c3c79a8f33028304d25de64#r73414955>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AJ3Bp36PHHhNN01t0zw4OmWwRui-mMBYks5qcPrZgaJpZM4JbqmZ>.
>


-- 
Tomas Mandys
Qore Technologies
Email: tomas.mandys@qoretechnologies.com
Tel: +420-604690589
Skype: tmandys@hotmail.com